src/deadline/job_attachments/_utils.py

```diff
@@ -98,3 +101,56 @@ def _is_windows_file_path_limit() -> bool:
 
         return bool(ntdll.RtlAreLongPathsEnabled())
     return True
+
+
+def _retry(
+    ExceptionToCheck: Union[Type[Exception], Tuple[Type[Exception], ...]] = AssertionError,
+    tries: int = 4,
+    delay: Union[int, float, Tuple[Union[int, float], Union[int, float]]] = 3.0,
+    backoff: float = 2.0,
```

Comment:
Nit: these defaults would lead to very slow failure in the worst case. Would it make sense to tighten them up?

Reply:
Sure, I can adjust them in a follow up (i have many PRs on going)

how about 1 retry, delay of 1s, back off 1s?